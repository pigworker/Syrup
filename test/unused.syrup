f(<Bit>) -> <Bit>
f(X) = X where

  Y = X
  Z = Y
  T = [X,Y,Z]