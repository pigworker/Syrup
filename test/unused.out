Warning: the wires T, Y, Z are defined but never used.

Circuit f is defined.

f(<Bit>) -> <Bit>
f(X) = X

